example_relay.conf

```diff
@@ -127,6 +127,16 @@ docker:
   # Default: 5m
   clean_interval: 5m
 
+  # Per container memory allocation (in megabytes)
+  # Environment variable: $RELAY_DOCKER_CONTAINER_MEMORY
+  # Default: 16
+  container_memory: 16
+
+  # Version of the command interface driver to use.
+  # See http://github.com/operable/circuit for details.
+  # Default: 0.8
+  command_driver_version: 0.8
```

Comment:
Is this intended to be exposed to the user? Can it me meaningfully changed? If not, can it be hard-coded somewhere?